packages/ui/src/app/pages/Forum/ForumThread.tsx

```diff
@@ -33,7 +34,18 @@ export const ForumThread = () => {
   const { active } = useMyMemberships()
 
   const sideNeighborRef = useRef<HTMLDivElement>(null)
+  const newPostRef = useRef<HTMLDivElement>(null)
   const history = useHistory()
+  const [replyTo, setReplyTo] = useState<ForumPost | undefined>()
+
+  const onReply = (post: ForumPost) => {
+    setReplyTo(post)
+    newPostRef.current?.scrollIntoView({ behavior: 'smooth', inline: 'end' })
```

Comment:
This seems to try to scroll to where the editor used to be, and because the quoted post appears next render, it doesn't scroll the editor fully into view. I tried using `useEffect` instead and it seemed to work.
```ts
  useEffect(() => {
    replyTo && newPostRef.current?.scrollIntoView({ behavior: 'smooth', inline: 'end' })
  }, [replyTo])
```